sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Release.java

```diff
@@ -19,15 +19,16 @@
  */
 package org.sonar.updatecenter.common;
 
+import com.google.common.base.Function;
+import com.google.common.base.Predicate;
+import com.google.common.collect.Collections2;
 import com.google.common.collect.ImmutableSortedSet;
 import org.apache.commons.lang.StringUtils;
 import org.apache.commons.lang.builder.ToStringBuilder;
 
-import java.util.Arrays;
-import java.util.Date;
-import java.util.Set;
-import java.util.SortedSet;
+import java.util.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2208)